src/main/java/uk/ac/ebi/atlas/home/HomeController.java

```diff
@@ -60,18 +61,20 @@ public String getHome(Model model) {
         model.addAttribute("species", allSpecies);
         model.addAttribute("speciesPath", ""); // Required by Spring form tag
 
-        model.addAttribute("numberOfStudies", experimentInfoListService.listPublicExperiments().size());
+        model.addAttribute("numberOfStudies", experimentTrader.getPublicExperiments().size());
 
         var numberOfSpecies =
-                experimentInfoListService.listPublicExperiments().stream()
-                        .map(ExperimentInfo::getSpecies)
+                experimentTrader.getPublicExperiments().stream()
```

Comment:
Is this a stale line from another story?

Reply:
No, since the `ExperimentInfoListService` is now gone, we need another way to get the public experiments.